app/src/org/commcare/android/tasks/ResourceEngineTask.java

```diff
@@ -27,82 +27,111 @@
 import org.javarosa.xml.util.UnfullfilledRequirementsException;
 import org.javarosa.core.services.Logger;
 
-import android.content.Context;
 import android.content.SharedPreferences;
 import android.content.SharedPreferences.Editor;
 import android.os.SystemClock;
 
 /**
- * This task is responsible for
- *
  * @author ctsims
- *
  */
-public abstract class ResourceEngineTask<R> extends CommCareTask<String, int[], org.commcare.android.tasks.ResourceEngineTask.ResourceEngineOutcomes, R> implements TableStateListener {
+public abstract class ResourceEngineTask<R>
+        extends CommCareTask<String, int[], org.commcare.android.tasks.ResourceEngineTask.ResourceEngineOutcomes, R>
+        implements TableStateListener {
 
     public enum ResourceEngineOutcomes implements MessageTag {
-        /** App installed Succesfully **/
+        /**
+         * App installed Succesfully
+         */
         StatusInstalled("notification.install.installed"),
 
-        /** Missing resources could not be found during install **/
+        /**
+         * Missing resources could not be found during install
+         */
         StatusMissing("notification.install.missing"),
 
-        /** Missing resources could not be found during install **/
+        /**
+         * Missing resources could not be found during install
+         */
         StatusMissingDetails("notification.install.missing.withmessage"),
 
-        /** App is not compatible with current installation **/
+        /**
+         * App is not compatible with current installation
+         */
         StatusBadReqs("notification.install.badreqs"),
 
-        /** Unknown Error **/
+        /**
+         * Unknown Error
+         */
         StatusFailUnknown("notification.install.unknown"),
 
-        /** There's already an app installed **/
+        /**
+         * There's already an app installed
+         */
         StatusFailState("notification.install.badstate"),
 
-        /** There's already an app installed **/
+        /**
+         * There's already an app installed
+         */
         StatusNoLocalStorage("notification.install.nolocal"),
 
-        /** Install is fine **/
+        /**
+         * Install is fine
+         */
         StatusUpToDate("notification.install.uptodate"),
 
-        /** Certificate was bad **/
+        /**
+         * Certificate was bad
+         */
         StatusBadCertificate("notification.install.badcert");
 
-        ResourceEngineOutcomes(String root) {this.root = root;}
+        ResourceEngineOutcomes(String root) {
+            this.root = root;
+        }
+
         private final String root;
-        public String getLocaleKeyBase() { return root;}
-        public String getCategory() { return "install_update"; }
+
+        public String getLocaleKeyBase() {
+            return root;
+        }
+
+        public String getCategory() {
+            return "install_update";
+        }
     }
 
-    Context c;

```

Comment:
Removed this unused var, also changing the class constructor to not take in a Context